articles/application-gateway/application-gateway-autoscaling-zone-redundant.md

```diff
@@ -72,26 +72,26 @@ Total price = $148.8 + $297.6 = $446.4
 
 **Example 2**
 
-An Application Gateway standard_v2 is provisioned for a month and during this time it receives 25 new SSL connections/sec, average of 8.88-Mbps data transfer. Assuming connections are short lived, your price would be:
```

Comment:
The base document seems slightly different from the current published document. Is this PR still valid today?

https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-autoscaling-zone-redundant
> Example 2
> An Application Gateway standard_v2 is provisioned for a month, with zero minimum instances, and during this time it receives 25 new TLS connections/sec, average of 8.88-Mbps data transfer. Assuming connections are short lived, your price would be: